swatch-picker/open-swatch-picker.js

```diff
@@ -0,0 +1,2204 @@
+                </button>
+                <span>Rose 2</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 3" style="background:var(--rose-3)">
+                    &nbsp;
+                </button>
+                <span>Rose 3</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 4" style="background:var(--rose-4)">
+                    &nbsp;
+                </button>
+                <span>Rose 4</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 5" style="background:var(--rose-5)">
+                    &nbsp;
+                </button>
+                <span>Rose 5</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 6" style="background:var(--rose-6)">
+                    &nbsp;
+                </button>
+                <span>Rose 6</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 7" style="background:var(--rose-7)">
+                    &nbsp;
+                </button>
+                <span>Rose 7</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 8" style="background:var(--rose-8)">
+                    &nbsp;
+                </button>
+                <span>Rose 8</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 9" style="background:var(--rose-9)">
+                    &nbsp;
+                </button>
+                <span>Rose 9</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 10" style="background:var(--rose-10)">
+                    &nbsp;
+                </button>
+                <span>Rose 10</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 11" style="background:var(--rose-11)">
+                    &nbsp;
+                </button>
+                <span>Rose 11</span>
+                </div>
+                
+                <div swatch="">
+                <button aria-label="Copy Rose 12" style="background:var(--rose-12)">
+                    &nbsp;
+                </button>
+                <span>Rose 12</span>
+                </div>
+                
+            </div>
+
+        </main>
+    </dialog>
+`
+export class OpenSwatchPicker extends HTMLElement {
+    static define(tagName = "open-swatch-picker") {
+        customElements.define(tagName, this)
+    }
+    shadowRoot = this.attachShadow({ mode: "open" });
+
+    constructor(){
+        super();
+        this._internals = this.attachInternals();
+    }
```

Comment:
Internals can be a static initializer too

```suggestion
    shadowRoot = this.attachShadow({ mode: "open" });
    #internals = this.attachInternals();
```

(You'll also need to update to `#internals` as a field name everywhere else).